plugins/vpc-eni/e2eTests/e2e_test.go

```diff
@@ -0,0 +1,301 @@
+//go:build e2e_test
+// +build e2e_test
+
+// Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//     http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package e2e
+
+import (
+	"context"
+	"fmt"
+	"net"
+	"os"
+	"strconv"
+	"syscall"
+	"testing"
+
+	"github.com/aws/amazon-vpc-cni-plugins/network/eni"
+	"github.com/containernetworking/cni/pkg/invoke"
+	"github.com/containernetworking/plugins/pkg/ns"
+	"github.com/containernetworking/plugins/pkg/testutils"
```

Comment:
About creating new dependencies to the "ns" and "testutils" packages: Does our own network/netns package in this repo not offer the same functionality? If not, could we add it to ours instead of creating a dependency on a 3rd party package?

Since these 3rd party packages are internal to CNI, we've seen changes in their signatures and implementation break our code before, hence avoid creating new dependencies if possible.

Reply:
Yeah network/netns does provide functions to do the same thing except that testutils package can create test network namespaces with random names. But I agree that there is not a strong reason to use new/internal packages from CNI. I will update to use the existing packages. Thank you!

Reply:
Updated in [ed8da834](https://github.com/aws/amazon-vpc-cni-plugins/pull/103/commits/ed8da834c06a5606b85bd5dc8ef6b8e93323c9fc). Diff is much cleaner now.